Competitive_Programming/Python/Greedy/activity_selection.md

```diff
@@ -0,0 +1,107 @@
+# Activity Selection Problem Using Greedy Approach
+
+An <b>activity selection</b> is the problem of scheduling a resource among several competing actvities.
+
+Here, we will use the <b>greedy approach</b> to address this problem.
+
+```
+Greedy Algorithm - It is an algorithmic strategy that makes the best optimal choice at each small stage which will eventually lead to an optimum solution for the entire problem. It always picks the best immediate output unlike dynamic approach which takes all the test cases into consideration.
+```
```

Comment:
Better you just highlight this sentance. The reader can't scroll to right till end, it's a bad user experience.